client/my-sites/email/email-providers-comparison/style.scss

```diff
@@ -202,3 +200,51 @@
 		}
 	}
 }
+
+.email-providers-comparison__gsuite-user-list-actions-container {
+	$container-top-margin: 1em;
+	$container-left-margin: 1em;
+	$child-top-margin: 1em;
+	$child-left-margin: 1em;
+
+	display: flex;
+	flex-direction: column;
+	margin-top: $container-top-margin;
+
+	@include breakpoint-deprecated( '>480px' ) {
+		flex-direction: row;
+		margin-top: 0;
+		margin-left: $container-left-margin;
+	}
+
+	@include breakpoint-deprecated( '>660px' ) {
+		flex-direction: column;
+		margin-left: 0;
+		margin-top: 0;
+	}
+
+	@include breakpoint-deprecated( '>800px' ) {
+		flex-direction: row;
+		margin-top: 0;
+		margin-left: $container-left-margin;
+	}
+
+	> * + * {
```

Comment:
I think we should use `.button + .button` here to specifically handle buttons.

Reply:
Will make the change but I wonder if that constraint _will_ always be satisfied (i.e, in case of icon buttons or if some other wrapper element is present like in a button group?)